test/externalTests/bleeps.sh

```diff
@@ -45,7 +45,7 @@ function bleeps_test
     local compile_only_presets=()
     local settings_presets=(
         "${compile_only_presets[@]}"
-        #ir-no-optimize            # Compilation fails with: "YulException: Variable expr_15509_mpos is 4 too deep in the stack". No memoryguard was present.
+        #ir-no-optimize            # Compilation fails with: "YulException: Variable expr_15509_mpos is 4 too deep in the stack". No memory guard was present.
```

Comment:
This is a copy of the error message with which the compilation fails, the spelling `memoryguard` is deliberate to reflect the `memoryguard` builtin. :)
Please revert this change.